src/commands/synthetics/README.md

```diff
@@ -682,7 +682,14 @@ These files take precedence over global configuration files, environment variabl
 Global Config < Environment variables < CLI parameters < Test Config
 ```
 
-By default, `datadog-ci` runs at the root of the working directory and looks for `{,!(node_modules)/**/}*.synthetics.json` files (every file ending with `.synthetics.json`, except for those in the `node_modules` folder) to find a test configuration file. This can be manually configured with the [`files` parameter](#files).
+To determine which tests to run, one or more of those options may be passed to `datadog-ci`:
+- The [`files` option](#files)
+- The [`publicIds` option](#publicids)
+- The [`testSearchQuery` option](#testsearchquery)
+
+If none of these options is passed, `datadog-ci` auto-discovers test configuration files with the `{,!(node_modules)/**/}*.synthetics.json` glob pattern (every file ending with `.synthetics.json`, except for those in the `node_modules` folder).
+
+**Note**: The file search starts from the current working directory, so it may be slow if the command is run from a big directory, like a home folder. For better performance, use the options above to specify the tests, or refine the glob pattern with the [`files` option](#files).
```

Comment:
```suggestion
**Note**: The file search starts from the current working directory, so it may be slow if the command is run from a big directory, like a home folder. For better performance, use the options above to specify the tests, or refine the path using the glob pattern with the [`files` option](#files).
```
nit: just to specify that narrowing down the path is the thing that would actually fix the issue with slowness.

Also can we add a sentence for the interaction between ids from `--public-id` or `--search` and the ids in a test config

Reply:
Not really, they could change it in a different way: for example, changing `**` to `*` would help. That's why I didn't want to be too specific, and using the word "refine" only means "make it more specific".

I'll try to find a better wording

Reply:
> Also can we add a sentence for the interaction between ids from `--public-id` or `--search` and the ids in a test config

I don't think it's useful to document this as it's an internal behavior, and hopefully we made the behavior good enough so that the user doesn't have to think about it